docs-src/docs/installation.md

```diff
@@ -34,7 +34,14 @@ npm install -g . # make "auspice" available globally
 ```
 
 Note that (at least on MacOS) this symlinks the source directory into the global `node_modules` folder so that changes to source code are automatically reflected in the globally available `auspice` command.
-We have had good success using a conda environment for development.
+
+To separate the npm-distributed and your development version of auspice, we recommend running Auspice inside of a virtual environment such as [Conda](https://conda.io/en/latest/).
```

Comment:
Good addition -- this is also detailed on the nextstrain.org docs: https://nextstrain.org/docs/getting-started/local-installation#install-auspice-from-source. Can you include this link here? (or just point users to those docs)